roles/ceph-defaults/defaults/main.yml

```
@@ -330,24 +330,28 @@ mon_host_v2:
 # CEPHFS #
 ##########
 cephfs: cephfs # name of the ceph filesystem
-cephfs_data: cephfs_data # name of the data pool for a given filesystem
-cephfs_metadata: cephfs_metadata # name of the metadata pool for a given filesystem
-
 cephfs_data_pool:
-  name: "{{ cephfs_data }}"
-  pgs: "{{ osd_pool_default_pg_num }}"
+  name: "cephfs_data"
```

@rishabh-d-dave what about something like this:

```suggestion
  name: "{{ cephfs_data if cephfs_data is defined else 'cephfs_data' }}"
```